api/helpers/uploadedFileErrorChecks.js

```diff
@@ -30,13 +30,17 @@ async function connectToClamAV(req) {
 }
 
 function initialiseClamScan(req) {
-    const { clamav_host: clamavHost, clamav_port: clamavPort } =
-        req._sails.config.upload;
-
+    const {
+        clamav_host: clamavHost,
+        clamav_port: clamavPort,
+        clamav_debug_enabled: clamavDebugEnabled,
+    } = req._sails.config.upload;
     const clamAvOptions = {
+        debugMode: JSON.parse(clamavDebugEnabled),
```

Comment:
You've declared clamav_debug_enabled in the bit above but then don't use it.
In this line you do a JSON.parse on the env variable but in the bit above you don't.

Reply:
Sorry, this is just me being confusing. On line **36** I'm renaming `clamav_debug_enabled` to `clamavDebugEnabled` purely for preference, the syntax is a bit weird for that. 

Similar to the way `clamav_host` has been renamed to `clamavHost` and used on line **42**.
```js
const { bar: foo } = { bar: 123 };
bar // Uncaught ReferenceError: bar is not defined
foo // 123
```